app/com/gu/floodgate/reindex/Progress.scala

```diff
@@ -0,0 +1,6 @@
+package com.gu.floodgate.reindex
+
+import play.json.extra.JsonFormat
+
+@JsonFormat
+case class Progress(status: String, documentsIndexed: Int, documentsExpected: Int)
```

Comment:
Could we make the status an enum/sealed trait? Or do you want to support arbitrary status messages? If you want to keep it as a string, could we at least make it something that looks a little more machine-readable than `"in progress"`?